wordle-web/src/components/LetterBase.vue

```diff
@@ -1,5 +1,19 @@
+<style>
+.charButton {
+  background: aqua;
+}
+</style>
+
 <template>
-  <v-btn :color="props.color">
+  <v-btn
+    class="charButton"
+    elevation="8"
+    min-size="x-small"
+    size="small"
+    height="50"
+    variant="outlined"
+    :color="props.color"
```

Comment:
You can apply a css gradient to the background-image or background css property.
You may need to use an "!important" if you choose to use a button variant that applies props.color to the background color of the button (like tonal). For some reason, when using a vuetify theme color, props.color will override any custom background css - so a !important is required. props.color actually returns the name of the theme color to use (not its hex value). 
https://developer.mozilla.org/en-US/docs/Web/CSS/gradient
https://www.w3schools.com/css/css3_gradients.asp

Example gradients:

radial-gradient(#ffffff40, #10000040)
linear-gradient(45deg, #ffffff40, #10000040)
linear-gradient(to right, red,orange,yellow,green,blue,indigo,violet)